x-pack/plugins/discover_enhanced/common/index.ts

```diff
@@ -5,7 +5,4 @@
  * 2.0.
  */
 
-// TODO: https://github.com/elastic/kibana/issues/110900
-/* eslint-disable @kbn/eslint/no_export_all */
-
-export * from './config';
+export { Config } from './config';
```

Comment:
:art: We could even just make this an `export type { Config } ..` instead.

Reply:
it's convinient to apply suche changes directly in vs code online!